### PR TITLE
Pre oop

### PR DESCRIPTION
**Pull request by usides**
### pre-oop

[Demo](https://usides.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/usides/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/usides/pre-oop/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+for (let item of [dog, cat, man, woman, catWoman]) {
```
Comment (madmaxWMFU) [inline submissions/usides/pre-oop/index.js:70]: Set this like a const
`const inhabitants = [dog, cat, man, woman, catWoman]`
Use the method `map` 
`inhabitants.map(object => .....)`
Write a function which returns a string of the object elements

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

File: submissions/usides/pre-oop/index.js (hunk 1 above)
Comment (usides) [reply submissions/usides/pre-oop/index.js]: Done

**Review by madmaxWMFU — APPROVED** [review]

**Comment by usides** [comment]
Dear mentors,
Can it be merged to main branch?

**Review by OleksiyRudenko — APPROVED** [review]
Well done!

**Pull request merged**